README.md

```diff
@@ -1,2 +1,129 @@
+
+`venueType` can be either `Pharmacy`, `Clinic`, `Community`, `Other`, `ClinicCommunity` or `ClinicPharmacy`
+
+## Test environments
+
+As the application is being developed, every Pull Request has its own test
+environment automatically built and deployed to.
+
+Every environment apart from the one we want the public to access requires
+basic authentication to access. The username and password are not secret, in
+fact they are included within environment variable table below.
+The intention with the authentication challenge is to prevent people whom may
+stumble across the site and not realise it is for testing, it also prevents
+access by search engines and other bots.
+
+## Environment variables
+
+Environment variables are expected to be managed by the environment in which
+the application is being run. This is best practice as described by
+[twelve-factor](https://12factor.net/config).
+
+In order to protect the application from starting up without the required
+env vars in place
+[require-environment-variables](https://www.npmjs.com/package/require-environment-variables)
+is used to check for the env vars that are required for the application to run
+successfully.
+This happens during the application start-up. If an env var is not found the
+application will fail to start and an appropriate message will be displayed.
+
+Environment variables are used to set application level settings for each
+environment.
+
+
+| Variable                           | Description                                                                                                 | Default                | Required |
+| :--------------------------------- | :---------------------------------------------------------------------------------------------------------- | ---------------------- | :------- |
+| `AZURE_STORAGE_CONNECTION_STRING`  | Azure storage connection string                                                                             |                        | yes      |
+| `CONTAINER_NAME`                   | Azure storage container name                                                                                | etl-output             |          |
+| `LOG_LEVEL`                        | [log level](https://github.com/trentm/node-bunyan#levels)                                                   | Depends on `NODE_ENV`  |          |
+| `NODE_ENV`                         | node environment                                                                                            | development            |          |
+| `SYNDICATION_API_KEY`              | API key to access syndication                                                                               |                        | yes      |
+| `API_URL`                          | URL to Syndicate service to scrape                                                                          | http://v1.syndication.nhschoices.nhs.uk/services/types/sexualhealthinformationandsupport | no      |
```

Comment:
Can `API_URL` be changed to provide some more detail about what it is please? Something like `SYNDICATION_SERVICE_END_POINT` or `SYNDICATION_SERVICE_URL`?